website/src/components/discourse/index.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
+import feedStyles from './styles.module.css';
+
+export default function DiscourseFeed({
+  order = 'latest_topic',
+  status = undefined,
```

Comment:
Likewise here, let's have status:solved as the default and override it where needed. 

OR, it might be easier to have a focused `DiscourseHelpFeed` which ultimately creates a `DiscourseFeed` but we don't have to put defaults into the main constructor. Is that practical?

Reply:
Set the default status to `solved`. We can create a more specific component which uses the original component with specific props if you'd like. If you feel like it would be easier for the team to work with we can get it setup. I would just want to go over all the settings that more specific component would need.

Reply:
Yeah I think it would be good to make a focused one. The default props: 
```javascript
order = 'latest_topic',
status = 'solved',
category = 'help',
link_text = 'Ask the Community',
link_href = // the same URL builder you've already got and I don't want to copy-paste
hide_cta = 'false'
```

Then on the main `DiscourseFeed`, everything would be undefined except for `hide_cta` which would be true. (Or, if you [changed it to be `show_cta`](https://github.com/dbt-labs/docs.getdbt.com/pull/2004#discussion_r974851260), then _everything_ could be undefined because `undefined` is falsy right?)